src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -87,6 +101,10 @@ const Body = BaseForm.extend({
     });
   },
 
+  doCustomURI () {
+    return request(this.customURI, 'POST', null, 3000);
```

Comment:
there are a couple of places that timeout is `3000`. can you add comments about why choose 3000?
if they have commonality, extra to an const variable.

Reply:
The custom timeout is only needed in loopback approach. In custom URI, we actually don't need to customize the timeout. I will remove the timeout from ajax call for custom URI approach.